### ISSUE TITLE
SES API Error: Local address contains illegal character in Mautic v3.2.0 

### ISSUE THREAD

**Issue by artem-ego** [mautic/mautic#9468]
## Bug Description
Sending emails using the AWS SES API causes the error in the ./var/logs/* like:
`[2020-12-02 16:43:34] mautic.ERROR: SES API Error: Local address contains illegal character [] []`
The Mautic says the email successfully sent. But the e-mail did not really send to the recipient.
The version of Mautic 3.2.0.
All works correctly if Mautic 3.2.0 configured to use SES-SMTP.
Downgrade Mautic to v3.1.2 solves the issue as it works with  SES-API and SES-SMTP correctly without errors with the same other configurations and recipient address.

| Q   | A
| --- | ---
| Mautic version |  3.2.0
| PHP version |  7.3.24

### Steps to reproduce
1. Configure Mautic 3.1.2 to use  SES-API 
2. Check all works correctly.
3. Upgrade Mautic to version 3.2.0
4. Try to send an e-mail
4. Check the logs file
 
### Log errors
[2020-12-02 15:55:33] mautic.ERROR: SES API Error: Local address contains illegal character [] []
[2020-12-02 16:07:09] mautic.ERROR: SES API Error: Local address contains illegal character [] []
[2020-12-02 15:21:04] mautic.ERROR: SES API Error: Local address contains illegal character [] []
[2020-12-02 15:33:20] mautic.ERROR: SES API Error: Local address contains illegal character [] []
[2020-12-02 16:43:34] mautic.ERROR: SES API Error: Local address contains illegal character [] []

**Comment by mabumusa1** [comment]
@artem-ego can you share the email address you send from, send to, and from name.

If they are private, please check if you have any unicode characters

It might be related to https://github.com/mautic/mautic/pull/9434

**Comment by artem-ego** [comment]
@mabumusa1 Sorry about the delay. No, my addresses don't have Unicode characters. All my Campaigns were broken after I have updated the version of Mautic from v3.1.2 to v3.2.0. I have used Mautic API to send some emails from my application and nothing to receive. 
Then I have tested via the "Send Email" button from the contact page, but I did not receive any e-mail too. I have changed the Mautic configuration to use SES SMTP and resent the test email from Mautic UI and via API and received all emails. So I reconfigured Mautic to SES API and retested and checked the logs and found the error messages. 
I had the backups of the Mautic home directory with the version v3.1.2. I just replaced the new Mautic folder with the old one and did not do any with my database. I opened the Mautic dashboard checked the version (3.1.2), checked the configuration (SES API), and repeated the tests. All works correctly without any errors in the logs files in the version of Mautic 3.1.2.
I have two instances of Mautic staging and production with different sets of contacts and both had got the same issues.

**Comment by mabumusa1** [comment]
@artem-ego thanks for the information, can you please share with me your configuration for the email.

Or better if you can make a loom video showing how you send the email and how did you configure it.

This issue is strange, I used the api multiple times and did not occur.

**Comment by RCheesley** [comment]
It may also be worth double checking things like database collation also?

**Comment by artem-ego** [comment]
Hello there sorry about the delay but I have used the Mautic 3.1.2 version which is works with SES-API without issues. But now I upgraded my version to 3.2.4 of Mautic for [security purposes](https://forum.mautic.org/t/security-release-for-all-versions-of-mautic-prior-to-2-16-5-and-3-2-3/17786) and got the same problem. I have not changed the database collation which is `utf8_unicode_ci` just updated Mautic to the latest version by running: `php bin/console mautic:update:apply`. 

Now I have found the reason for the problem: the field "Name to send mail as" in the top left in my settings contained the symbol ":" which caused the issue. After I deleted the ":" the SES-API works correctly in the Mautic version 3.2.4. It is weird because I never changed this field before and all Mautic versions up to 3.2.0. worked without issue with the same value.
It might be worth checking out as it could cause some problems in the future.
Thanks for your help!
